bundle/manifests/ibm-mongodb-operator.clusterserviceversion.yaml

```diff
@@ -298,7 +298,8 @@ spec:
           - challenges
           - issuers
           verbs:
-          - '*'
+          - create
```

Comment:
```
          - certmanager.k8s.io
          resources:
          - certificates
          - issuers
          verbs:
          - create
```

I believe the MongoDB operator only creates `certificates` and `issuers`. We don't need the `use` verb as well.

Reply:
fixed